src/studio/kdb/Config.java

```diff
@@ -25,16 +25,30 @@
 public class Config {
     private static final Logger log = LogManager.getLogger();
 
+    public enum FontKind {
+        EDITOR,
+        TABLE,
+        ;
+
+        String property() {
+            if (this == EDITOR)
+                return "font";
+            return name().toLowerCase() + "font";
+        }
+    }
+
```

Comment:
I would prefer to implement similar to how introduce bounds below. That is to have something like this 
```
public static final String EDITOR_FONT = configDefault("font", ConfigType.FONT, new Font(...)); // Monospaced is default name
public static final String GRID_FONT = configDefault("tableFont", ConfigType.FONT, getDefaultTableFont());
```
Is JTable default font the same in all L&F ? I don't know - i would think it depends on L&F. I would make an attempt to default GRID_FONT to what it is now for a user (for the L&F he/she selected).